.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
```

Comment:
Codacy found an issue: [Always tag the version of an image explicitly](https://app.codacy.com/manual/lucaslarson/HQ9/pullRequest?prid=6287850)